libs/ui/src/components/NeoButton/NeoButton.scss

```diff
@@ -47,6 +48,14 @@
       &.o-btn--disabled {
         @apply opacity-unset bg-k-shade border-k-grey text-k-grey;
       }
+
+      &.shiny:not(:hover):not(:disabled) {
+        @include shineEffect(var(--k-accent-light-3), lightgrey, false);
+
+        &:hover {
+            color: var(--k-accent2) !important;
```

Comment:
Could you try with it? `@apply hover:text-k-accent2`

Reply:
updated